index.js

```diff
@@ -8,6 +8,8 @@ export const setCookie = (name, value, options) => {
   const optionsWithDefaults = {
     days: 7,
     path: '/',
+    samesite: 'none'
+    secure: false,
```

Comment:
sorry havent looked at this repo in too long, but I'm making updates now. However, I think for this we might want to default to the browser default `samesite: 'lax'` https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#lax

Reply:
I'll just update it, just wanted to let you know

Reply:
OK, Good to to see you back :) Thank you. I missed that `lax` started to be default